src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -465,4 +465,7 @@ For details on valid range queries, please refer to Search Matching section in C
     <value>Specified PartitionName '{0}' does not exist.</value>
     <comment>{0} PartitionName value.</comment>
   </data>
+  <data name="DataPartitionFeatureCannotBeDisabled" xml:space="preserve">
+    <value>Data partitions feature cannot be disabled when the service has existing data.</value>
```

Comment:
nit:
```suggestion
    <value>Data partitions cannot be disabled while existing data has already been partitioned.</value>
```

Is there also any way for users to disable it? If they deleted all of their data / partitions, then could they disable the feature? If so, perhaps the error should indicate this.

Reply:
Currently there is no way to delete all the partitions unless they have access to the DB. But in future may be.